votor/src/certificate_pool.rs

```diff
@@ -511,12 +490,13 @@ impl CertificatePool {
 
     fn add_certificate(
         &mut self,
+        root_slot: Slot,
```

Comment:
this should not care about the root at all, but I guess that will be part of #368